apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -4,13 +4,15 @@ import {BodyTwoText, Heading1} from '@cdo/apps/componentLibrary/typography';
 import LearningGoalItem from './LearningGoalItem';
 import Button from '@cdo/apps/templates/Button';
 
-export default function RubricsContainer({unit, lesson}) {
+// TODO: After the MVP is done, consider how to handle stand-alone lessons/units
```

Comment:
I think this should work for standalone units? And standalone lessons don't exist right now so no need to worry about them!